### PR TITLE
Fixes #1189: ProfileChooser-FirstTime&Regular - Tablet (Landscape) (Highfi)

### LINKED ISSUE #1137: ProfileChooser - Tablet (Landscape) (Lowfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/6fb19f15-699a-40aa-8392-cdfc6d7bf8c5/L-Profile-Chooser-Regular

Implement low-fi UI for **ProfileChooser** tablet landscape mode

**Target PR date**: 3 June 2020
**Target completion date**: 6 June 2020

### LINKED ISSUE #1138: ProfileChooser - Tablet (Landscape) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/33741ea7-89da-4e5d-b2a4-9872f6db48b4/

Implement high-fi UI for **ProfileChooser** tablet landscape mode

**Target PR date**: 12 June 2020
**Target completion date**: 16 June 2020

### LINKED ISSUE #1189: ProfileChooser-FirstTime - Tablet (Landscape) (Highfi)
Mocks: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/33741ea7-89da-4e5d-b2a4-9872f6db48b4/L-Profile-Chooser-First-Time

Implement high-fi UI for **ProfileChooser-FirstTime** tablet landscape mode

**Target PR date**: 12 June 2020
**Target completion date**: 16 June 2020

### PR DESCRIPTION
**Pull request by MohamedMedhat1998**
## Note
The branch name should be `profile-chooser-ft-tablet-land-highfi` not `profile-chooser-ft-tablet-land-lowfi`, **highfi** not **lowfi**.  
Sorry about that mistake.
## Explanation

- Implemented High-fi UI for ProfileChooser-FirstTime & ProfileChooser-Regular tablet landscape
- Mock: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/screen/33741ea7-89da-4e5d-b2a4-9872f6db48b4/
- Fixes #1189 
- Fixes #1137 
- Fixes #1138 
## Screenshots (1024x768) emulator

screenshot-2020-06-15_14 14 41 943
screenshot-2020-06-15_14 16 53 798
screenshot-2020-06-15_14 17 22 518
screenshot-2020-06-15_14 19 14 926
screenshot-2020-06-15_14 19 23 869

## Screenshots (2560x1800) emulator

screenshot-2020-06-15_14 09 39 519
screenshot-2020-06-15_14 05 27 984
screenshot-2020-06-15_14 07 49 951

### Update
Updated the max width of `add_profile_description_text` `TextView` so that the sentence appears in two lines instead of one line.
screenshot-2020-06-16_18 49 25 233

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Review by rt4914 — CHANGES_REQUESTED** [review]
@MohamedMedhat1998 As discussed in the chat, these PRs should be done like this:
PR 1 - Entire landscape mode for regular + first-time.
PR 2 - Entire portrait mode for regular + first-time.


This is needed because of the interdependency of `first-time` and `regular` code and it is not possible to review half part because even if we do that in later stage again we will need to review both parts because there might be some changes. 

So I suggest including the `regular` view for landscape mode in this PR to make things work correctly.

Thanks.

**Comment by MohamedMedhat1998** [comment]
> 
> 
> @MohamedMedhat1998 As discussed in the chat, these PRs should be done like this:
> PR 1 - Entire landscape mode for regular + first-time.
> PR 2 - Entire portrait mode for regular + first-time.
> 
> This is needed because of the interdependency of `first-time` and `regular` code and it is not possible to review half part because even if we do that in later stage again we will need to review both parts because there might be some changes.
> 
> So I suggest including the `regular` view for landscape mode in this PR to make things work correctly.
> 
> Thanks.

@rt4914 Ok, I'll be working on it asap.  
Thanks!

File: app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml]: As per the mocks there should be marginEnd = 24dp too when first time chooser view is show. 

This is also make sure that the description text in looks good in 2 lines in this image:
https://user-images.githubusercontent.com/8533363/84657328-695b0180-af14-11ea-94b0-12228c9eab07.png

File: app/src/main/res/layout-sw600dp-land/profile_chooser_profile_view.xml
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/profile_chooser_profile_view.xml]: Ditto as `add_view` comment.

**Review by rt4914 — CHANGES_REQUESTED** [review]
LGTM, one nit change suggested.

**Comment by MohamedMedhat1998** [comment]
@mschanteltc Hi Chantel. Please take a look at the screenshots here that contain 5 items in a row.  
They are from a larger emulator (2560x1800), so it will be better to make the row take 5 items instead of 4.  
Rajat reviewed these screens and approved them visually. We are just waiting for your approval.  
Do they look good or there should be some change to be made?  
Thanks!

File: app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml]: Done!

File: app/src/main/res/layout-sw600dp-land/profile_chooser_profile_view.xml
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-land/profile_chooser_profile_view.xml]: Done!

**Comment by mschanteltc** [comment]
> @mschanteltc Hi Chantel. Please take a look at the screenshots here that contain 5 items in a row.
> They are from a larger emulator (2560x1800), so it will be better to make the row take 5 items instead of 4.
> Rajat reviewed these screens and approved them visually. We are just waiting for your approval.
> Do they look good or there should be some change to be made?
> Thanks!

These look great to me, awesome work Mohamed!

File: app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml]: There is one thing that I missed las time, as you are using padding, your dimen values should be `padding_24` or you should use `marginStart` and `margin_24`.
I think you should use `margins` here instead of `padding`

Comment (rt4914) [reply app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml]: Ditto for other file.

File: app/src/main/res/layout-sw600dp-land/profile_chooser_fragment.xml
Comment (rt4914) [inline app/src/main/res/layout-sw600dp-land/profile_chooser_fragment.xml]: Remove this tag

**Review by rt4914 — CHANGES_REQUESTED** [review]
Some final nit changes.

File: app/src/main/res/layout-sw600dp-land/profile_chooser_fragment.xml
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-land/profile_chooser_fragment.xml]: Done

File: app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml
Comment (MohamedMedhat1998) [reply app/src/main/res/layout-sw600dp-land/profile_chooser_add_view.xml]: Done

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
